.github/CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 /.config/                       @aspnet/build
 /build/                         @aspnet/build
 /eng/                           @aspnet/build
+/eng/common/                    @dotnet-maestro-bot
```

Comment:
Suggest including my, myself or I for the same reason I'm included on the next two lines